### PR TITLE
Now that the D3 is working, fine-tuning it for Attorney filtering

### PR DESCRIPTION
**Pull request by cjekal**
Adding the ability to filter by Attorney using the Attorney Search.

### CODE REVIEW

File: GetData/getdata.js [hunk 1]
```diff
@@ -21,32 +21,25 @@
       console.log("worksheets:");
       console.table(worksheets);
 
-      // Find a specific worksheet
-      var worksheet = worksheets.find(function (sheet) {
-        return sheet.name === "Person";
-        console.log("sheet.name:", sheet.name)
-      });
-
```
Comment (cjekal) [inline GetData/getdata.js:29]: this is no longer necessary

File: GetData/getdata.js [hunk 2]
```diff
@@ -21,32 +21,25 @@
...
-        let dataJson;
-        data.data.map(d => {
-          dataJson = {};
-          dataJson[data.columns[0].fieldName] = d[0].value; //1st column
-          dataJson[data.columns[1].fieldName] = d[1].value; //2nd column
-          dataJson[data.columns[2].fieldName] = d[2].value; //3rd column
-          dataArr.push(dataJson);
-        })
-        console.log("dataArr", dataArr);
+        let dataArr = processDataTable(data);
```
Comment (cjekal) [inline GetData/getdata.js:31]: refactored this to the `processDataTable` method

File: GetData/getdata.js [hunk 3]
```diff
@@ -21,32 +21,25 @@
...
         buildGraph(dataArr);
       });
 
+      erData.addEventListener(tableau.TableauEventType.FilterChanged, handler => {
+        console.log("filter changed!", handler);
+        handler.worksheet.getSummaryDataAsync().then(data => {
+          let dataArr = processDataTable(data);
+          buildGraph(dataArr);
+        });
+      });
```
Comment (cjekal) [inline GetData/getdata.js:41]: this wires the drawing of the d3 vis/graph to the filtering event.

**Review by cjekal — COMMENTED** [review]
@echasin please review!

**Comment by echasin** [comment]
Reviewed...

**Pull request merged**